docs/knowledge_base/tips/css_style_kit/index.md

```diff
@@ -0,0 +1,76 @@
+
+Prepare for a big change in your Taipy applications! 
+The new CSS Style Kit is here and it will make your application look even better. 
+It comes with pre-made styles, so your application will look great without you having to do much.
+
+But that's not all – you can also make your application look even nicer with some customization.
+
+![CSS Style Kit](css_style_kit.png){width=100%}
+
+The CSS Style Kit has some great features:
+
+- It has ready-made styles for Taipy elements.
+- You can change things like color or spacing easily by modifying CSS variables.
+- There are also utility CSS classes to make elements look more appealing.
+
+By default, the CSS Style Kit is turned on, so it affects how your page looks. 
+If you want to turn it off, you can do so by setting the CSS Style Kit parameter of the Gui.run() method to False.
+
+## Main CSS Variables
+
+The CSS Style Kit provides a long list of variables that affect how your Taipy application pages look. 
+It also offers predefined CSS classes for things like changing text style, 
+adding space around elements, adjusting how things are displayed, 
+and making elements more or less see-through.
+
+If these terms sound confusing, don't worry. We've made it simple for you to change how your text looks. 
+You can easily add color or center it.
+
+Now, let's use it in our application:
+
+```python linenums="1"
+&lt;|text-center| Taipy **App**{: .color-primary} |&gt;
+ 
+or
+ 
+Taipy **App**{: .color-primary}
+{: .text-center}
+```
+
+Let’s apply it to our application.
+
+![Main CSS Variables](css_style_kit_2.png){width=100%}
+
+The CSS Style Kit also offers styled sections like containers, cards, headers, and sidebars. 
+These can be used to make certain parts of your pages more noticeable or to control their size and position.
+
+For instance, you can use a container to add some space around your Markdown content.
+
+```python linenums="1"
```

Comment:
If the lines are not referred to, we don't need to specify the line numbers;
```suggestion
```python
```